Tasks/DownloadPipelineArtifactV2/task.json

```diff
@@ -0,0 +1,229 @@
+                "specificBuildWithTriggering"
+            ],
+            "type": "boolean",
+            "label": "When appropriate, download artifacts from the triggering build.",
+            "defaultValue": "false",
+            "required": false,
+            "visibleRule": "source == specific",
+            "helpMarkDown": "If checked, this build task will try to download artifacts from the triggering build. If there is no triggering build from the specified pipeline, it will download artifacts from the build specified in the options below."
+        },
+        {
+            "name": "runVersion",
+            "aliases": [
+                "buildVersionToDownload"
+            ],
+            "type": "pickList",
+            "label": "Build version to download",
+            "defaultValue": "latest",
+            "visibleRule": "source == specific",
+            "required": true,
+            "options": {
+                "latest": "Latest",
+                "latestFromBranch": "Latest from specific branch and specified Build Tags",
+                "specific": "Specific version"
+            }
+        },
+        {
+            "name": "runBranch",
+            "aliases": [
+                "branchName"
+            ],
+            "type": "string",
+            "label": "Branch name",
+            "defaultValue": "refs/heads/master",
+            "visibleRule": "source == specific && runVersion == latestFromBranch",
+            "required": true,
+            "helpMarkDown": "Specify to filter on branch/ref name, for example: ```refs/heads/develop```."
+        },
+        {
+            "name": "runId",
+            "aliases": [
+                "pipelineId",
+                "buildId"
+            ],
+            "type": "pickList",
+            "label": "Build",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "source == specific && runVersion == specific",
+            "properties": {
+                "EditableOptions": "True",
+                "DisableManageLink": "True"
+            },
+            "helpMarkDown": "The build from which to download the artifacts."
+        },
+        {
+            "name": "tags",
+            "type": "string",
+            "label": "Build Tags",
+            "defaultValue": "",
+            "visibleRule": "source == specific && runVersion != specific",
+            "required": false,
+            "helpMarkDown": "A comma-delimited list of tags. Only builds with these tags will be returned."
+        },
+        {
+            "name": "artifact",
+            "aliases": [
+                "artifactName"
+            ],
+            "type": "pickList",
+            "label": "Artifact name",
+            "defaultValue": "",
+            "required": false,
+            "properties": {
+                "EditableOptions": "True",
+                "DisableManageLink": "True"
+            },
+            "helpMarkDown": "The name of the artifact to download."
```

Comment:
>" [](start = 66, length = 1)

"The name of the artifact to download. If left empty, it will download all artifacts associated to the pipeline run."